routes/user-routes.go

```diff
@@ -0,0 +1,13 @@
+package routes
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/maayarosama/Blogging_system/controllers"
+)
+
+var RegisterUserStoreRoutes = func(router *mux.Router, c *controllers.Controller) {
```

Comment:
this should be refactored imo to

App has a router, logger, db connection, and to register a handler should be 

```
func (a *App) RegisterHandlers() {
	a.router.HandleFunc ...
}
```